magnum_cluster_api/resources.py

```diff
@@ -2556,6 +2564,12 @@ def get_object(self) -> objects.Cluster:
                                     self.cluster, "keystone_auth_enabled", True
                                 ),
                             },
+                            {
+                                "name": "controlPlaneAvailabilityZones",
+                                "value": utils.get_cluster_label(
```

Comment:
We've moved away from `get_cluster_label` because Magnum natively copies in all the labels to `cluster.labels`, so you can just `cluster.labels.get("...", "")` at this point.

Reply:
Done